notify/impl.go

```diff
@@ -68,14 +93,27 @@ func Build(confs []*config.Receiver, tmpl *template.Template) map[string]Fanout
 			if len(res) == 0 {
 				return nil
 			}
-			return n.Notify(ctx, res...)
+
+			var (
+				integr = n.name()
+				// The receiver is of the form <name>/<integration>/<num>.
+				// We only care about the first component.
+				recv = strings.Split(receiver(ctx), "/")[0]
```

Comment:
Currently not. It's sort of dirty in general restoring the info from a string that should be opaque – I might have to think about some tweaks elsewhere to do this better.